src/twisted/newsfragments/9585.feature

```diff
@@ -0,0 +1 @@
+twisted.internet._sslverify.ClientTLSOptions now supports validating IP addresses from the certificate subjectAltName
```

Comment:
The changelog should be written in terms of public interfaces, in this case `twisted.internet.ssl.optionsForClientTLS` instead of `twisted.internet._sslverify.ClientTLSOptions`.